Documentation/freebsd-instructions.md

```diff
@@ -0,0 +1,200 @@
+Build CoreCLR on FreeBSD
+======================
+
+This guide will walk you through building CoreCLR on FreeBSD and running Hello World.  We'll start by showing how to set up your environment from scratch.
+
+Environment
+===========
+
+These instructions are written assuming the 10.1-RELEASE, since that's the release the team uses.
+
+These instrutions assume you use the binary package tool `pkg` to install the environment. Compiling the dependencies from src using the ports tree might work too, but is untested.
```

Comment:
"instrutions" -> "instructions"

Is "pkg" the analog of apt-get for FreeBSD? For people w/o FreeBSD knowledge (like me), it would be nice to clarify its role.